docs/ops.html

```diff
@@ -766,27 +766,9 @@ <h5 class="anchor-heading"><a id="georeplication-flow-configure" class="anchor-l
 </code></pre>
 
   <p>
-    Additional configuration settings are supported, some of which are listed below. In most cases, you can leave these settings at their default values. See <a href="https://github.com/apache/kafka/blob/trunk/connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMakerConfig.java">MirrorMakerConfig</a> and <a href="https://github.com/apache/kafka/blob/trunk/connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java">MirrorConnectorConfig</a> for further details.
+    Additional configuration settings are supported which can be left with their default values in most cases. See <a href="https://kafka.apache.org/documentation/#mirrormakerconfigs">MirrorMaker Configs</a>.
```

Comment:
Nit: we can just use a relative URL here
```suggestion
    Additional configuration settings are supported which can be left with their default values in most cases. See <a href="/documentation/#mirrormakerconfigs">MirrorMaker Configs</a>.
```